src/Tools/dotnet-user-jwts/src/Helpers/Jwt.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+public record Jwt(string Id, string Name, string Audience, DateTimeOffset NotBefore, DateTimeOffset Expires, DateTimeOffset Issued, string Token)
+{
+    public IEnumerable<string> Scopes { get; set; } = new List<string>();
+
+    public IEnumerable<string> Roles { get; set; } = new List<string>();
+
+    public IDictionary<string, string> CustomClaims { get; set; } = new Dictionary<string, string>();
+
+    public override string ToString() => Token;
+
+    public static Jwt Create(
+        JwtSecurityToken token,
+        string encodedToken,
+        IEnumerable<string> scopes = null,
+        IEnumerable<string> roles = null,
+        IDictionary<string, string> customClaims = null)
+    {
+        return new Jwt(token.Id, token.Subject, token.Audiences.FirstOrDefault(), token.ValidFrom, token.ValidTo, token.IssuedAt, encodedToken)
```

Comment:
```suggestion
        return new Jwt(token.Id, token.Subject, token.Audiences.First(), token.ValidFrom, token.ValidTo, token.IssuedAt, encodedToken)
```

There should always be at least one audience, right?

Reply:
It can be empty if there's [no audience set](https://docs.microsoft.com/en-us/dotnet/api/system.identitymodel.tokens.jwt.jwtsecuritytoken.audiences?view=azure-dotnet#remarks). For the CLI scenario, this will translate to situations where the user doesn't define an HTTPS-based URL in launch settings. At runtime, this means that the audience of the token [will not be valdiated](https://docs.microsoft.com/en-us/dotnet/api/microsoft.identitymodel.tokens.tokenvalidationparameters.validateaudience?view=azure-dotnet).